### ISSUE TITLE
Canonical docs repo for an organization

### ISSUE THREAD

**Issue by mehulkar** [isaacs/github#909]
If I have "my-org", it would be great to have the "my-org/docs" be rendered as a github page at `pages.github.com/my-org`.

Organizing repos inside an organization is awesome, but it is often associated with documentation / wikis that applies to ALL the repos inside it. Although I can setup `my-org/docs` with gh-pages, the URL then becomes `pages.github.com/my-org/docs` This feels a little wrong.

**Comment by cirosantilli** [comment]
doesnt the magic `orgname.github.io` reponame work for orgs as well as users (no idea)

**Comment by strugee** [comment]
Yes it does - it's called an "organization page" as opposed to a "user page" but AFAIK it's basically the same thing

**Comment by mehulkar** [comment]
An org page is sort of ok. but it's not a wiki to be viewed Github for developers, it's more of a customer facing thing. Secondly, org pages do not work on GH Enterprise in that format, as far as I know.

**Comment by strugee** [comment]
@AlecBenzer FWIW the way I've seen this done before is to just have a repo called "wiki" for the org. Then the "wiki" repo's, well, wiki, is used for the whole org.

**Comment by drewnoakes** [comment]
@strugee I proposed some ideas in #1021.

**Comment by mehulkar** [comment]
> I don't think pages.github.com/my-org makes much sense as opposed to the already-existing my-org.github.io,

subdomains don't work with GH Enterprise. 

But I have a workaround. If i name my repo: `<owner>.<hostname>`, then GH pages will render at `pages.<hostname>/<owner>`, which is a good enough for me.
